### PR TITLE
MAINT: Minor tweaks to the base code

### PR DESCRIPTION
**Pull request by pastewka**
- MAINT: Fined-grained enabling of tabnav tabs in settings file
- MAINT: Moved `allauth`-dependent views into `views_allauth`
- MAINT: Added license information to version information modal
- MAINT: Upgrade of SurfaceTopograpy to 1.3.3 (#950)
- CI: Switched from Travis CI to GitHub Action

### CODE REVIEW

**Review by mcrot — CHANGES_REQUESTED** [review]
Code looks good overall. One question: The package "collectfast" is no longer used
in production. Is this intended?

And, when checking the branch in my topobank-stack-devel enviroment, I have the following problems:

- the datepicker is not working (what is the CI problem with version 5?, I'll also try)
- the static files are not found any more

The latter persists after successfully calling "collectstatic".
It is probably related to the removal of whitenoise, but not sure yet.

**Comment by mcrot** [comment]
`django-bootstrap-datepicker-plus==5.0.2` works without problems during build, probably also 5.0.3, which is the newest.

**Comment by pastewka** [comment]
You need the following update to the development stack: https://github.com/ContactEngineering/topobank-stack-development/pull/12

You don't need `whitenoise` in development. You also don't need to call `collectstatic` in development. Both are for production. `collectfast` would therefore also just affect production, and I don't really see why we need it.

The datepicker works for me, but I've updated the requirements to >=4.0.

File: setup.cfg [hunk 1]
```diff
@@ -177,7 +177,7 @@ install_requires =
     pygments>=2.7.4
 
     # datepicker
-    django-bootstrap-datepicker-plus<5.0
```
[the remark this answers is not in the document]
Comment (mcrot) [reply setup.cfg]: Yes, it's better to pin with the requirement files.

**Comment by mcrot** [comment]
The problem was that I had `DEBUG=no` in my config for some reason - this disabled the automatic delivery of static files as described [here](https://docs.djangoproject.com/en/4.1/howto/static-files/#serving-static-files-during-development).
With `DEBUG=yes` it's working fine.

**Pull request merged**